drivers/sensor/hx711/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_HX711 hx711.c)
```

Comment:
Please just use `zephyr_library_sources(hx711.c)`. This style is redundant and is undoing the cleanup work that was done in bd8afe9365e19d579417a514dd3bc87ee1667b9f

Reply:
Done.